archivist_samples/software_bill_of_materials/README.md

```diff
@@ -18,24 +18,35 @@ Install the [RKVST Python package](https://pypi.org/project/jitsuin-archivist/ "
 Get an authorization bearer token and store it in the file `.auth_token`. If you don't know how to do this, please refer to the [RKVST documentation](https://docs.jitsuin.com/docs/setup-and-administration/getting-access-tokens-using-client-secret/ "Getting an auth token")
```

Comment:
usually the authtoken file is in a subdirectory'credentials which disallows group and root access

Reply:
Accepted, will change

Reply:
Fixed in update 5